ui/view/view.go

```diff
@@ -28,7 +28,7 @@ func (v *View) Set(param string, value interface{}) *View {
 
 // Render executes the template with arguments.
 func (v *View) Render(template string) []byte {
-	return v.tpl.Render(template+".html", request.UserLanguage(v.r), v.params)
+	return v.tpl.Render(template+".html", v.params)
```

Comment:
The point of this is to avoid duplicating logic for selecting the default language. I'm not very familiar with go, so I'm not sure about the implication of pulling data out of `interface{}` like this.